tonic_validate/classes/benchmark.py

```diff
@@ -7,6 +7,19 @@
 
 @dataclass
 class BenchmarkItem:
+    """
+    A benchmark item is a question and an optional answer used in a benchmark.
```

Comment:
A benchmark item is a question that is used in a benchmark. It can include an optional answer.